admin/edit-user.html

```diff
@@ -3,10 +3,112 @@
 title: Add user, edit user, your account
 ---
 
-<div class="mw6 center mv6">
-  <h1>This will show states for:</h1>
-  <ul>
-    <li>Add user (only available to Admin user)</li>
-    <li>Edit user (only available to Admin user)</li>
-  </ul>
-</div>
+<main class="mw7 center ph2">
```

Comment:
I'm thinking it might be better to just make the main container mw6 instead of mw7, so then we don't have all the extra space on the right and don't need to define any widths or use any flexbox. What do you think?

Reply:
Updated to mw6